### PR TITLE
[ZT] move user logout information out of FAQ

### PR DESCRIPTION
**Pull request by ranbel**
pcx-4496

### CODE REVIEW

File: content/cloudflare-one/faq/teams-authentication-faq.md [hunk 1]
```diff
@@ -15,15 +15,3 @@ Yes. Your team can simultaneously use multiple providers, reducing friction when
 ## What if the identity provider my team uses is not listed?
 
 You can add your preferred identity providers to Cloudflare Access even if you do not see them listed on the Zero Trust dashboard, as long as these providers support SAML 2.0 or [OpenID Connect (OIDC)](/cloudflare-one/identity/idp-integration/generic-oidc/).
-
-## How do end users log out of an application protected by Access?
```
Comment (jroyal) [inline content/cloudflare-one/faq/teams-authentication-faq.md:19]: tbf this is a popular faq :)

File: content/cloudflare-one/identity/users/session-management.md [hunk 2]
```diff
@@ -79,6 +79,26 @@ The user will no longer be able to log in to any application protected by Access
...
+
+To force log out of an Access application, go to:
+
+`<your-application-domain>/cdn-cgi/access/logout`
+
+This immediately clears the authorization cookie from the user's browser. All previously issued application tokens will stop being accepted in 20-30 seconds.
+
+### Log out of all applications
+
+To log out of all Access applications, go to:
```
Comment (ranbel) [inline content/cloudflare-one/identity/users/session-management.md]: It's unclear what happens when you go to `<your-team-name>.cloudflareaccess.com/cdn-cgi/access/logout`.

File: content/cloudflare-one/identity/users/session-management.md [hunk 3]
```diff
@@ -79,6 +79,26 @@ The user will no longer be able to log in to any application protected by Access
...
+
+`<your-team-name>.cloudflareaccess.com/cdn-cgi/access/logout`
+
+This immediately clears the authorization cookie from the user's browser. All previously issued global session tokens will stop being accepted in 20-30 seconds.
```
Comment (jroyal) [inline content/cloudflare-one/identity/users/session-management.md]: Logout will revoke all of the requesting users tokens across that account. We don't have a way to differentiate logouts between a single application and all applications right now.

**Review by jroyal — CHANGES_REQUESTED** [review]

File: content/cloudflare-one/faq/teams-authentication-faq.md (hunk 1 above)
Comment (ranbel) [reply content/cloudflare-one/faq/teams-authentication-faq.md:19]: Good to know. I've added it back in.

**Review by jroyal — APPROVED** [review]

File: content/cloudflare-one/identity/users/session-management.md (hunk 3 above)
Comment (ranbel) [reply content/cloudflare-one/identity/users/session-management.md]: Wdyt about the new wording?

**Pull request merged**